sound/soc/intel/boards/Makefile

```diff
@@ -42,7 +42,7 @@ snd-soc-sof-sdw-objs += sof_sdw.o				\
 			sof_sdw_rt711.o sof_sdw_rt_sdca_jack_common.o	\
 			sof_sdw_rt712_sdca.o sof_sdw_rt715.o	\
 			sof_sdw_rt715_sdca.o sof_sdw_dmic.o	\
-			sof_sdw_hdmi.o
+			sof_sdw_cs42l42.o sof_sdw_hdmi.o
```

Comment:
This is still not two lines as I asked... Please be a bit more careful with your commits, it feels like Ground Hog Day where  I have to repeat the same thing...

Reply:
done